src/constants/common.ts

```diff
@@ -17,13 +20,13 @@ export const GENERIC_ERROR_MESSAGE = `An error occurred. Please get in touch wit
 
 export const AP_LOGO: ImageProps = {
   href: "https://angelprotocol.io/",
-  src: angelProtocolLogoWht,
+  src: IS_AST ? angelProtocolLogoWht : angelGivingLogoWht,
```

Comment:
@SovereignAndrey can you confirm this is expected?

Reply:
@misicnenad confirmed this with andrey before implementing:
![image](https://user-images.githubusercontent.com/23124046/231673584-dde5c0d3-e7ac-4389-8581-6014417f083d.png)